### PR TITLE
fix(web): various ui fixes releated to onboarding, registration verification, deployment list

### PR DESCRIPTION
**Pull request by robot9706**
- Fixed email resend not working on the verify page
- Fixed deployment list not updating after deleting a deployment
- Fixed error popping up when trying to hide onboarding tips
- Fixed onboarding tips enabling after a login-logout
- Added "Log out" option to the verify page
- Improved onboarding query

### CODE REVIEW

File: web/crux/src/services/kratos.service.ts [hunk 1]
```diff
@@ -120,25 +120,48 @@ export default class KratosService {
...
   }
 
   async disableOnboarding(userId: string): Promise<void> {
+    const identity = (
+      await this.identity.getIdentity({
+        id: userId,
+      })
+    ).data
+
+    // Due to a bug in kratos (https://github.com/ory/kratos/pull/3304)
```
Comment (polaroi8d) [inline web/crux/src/services/kratos.service.ts:153]: This is fixed in here: https://github.com/ory/kratos/issues/3244 - so is this still relevant?

**Comment by ngyradi** [comment]
tested, works locally

File: web/crux/src/services/kratos.service.ts (hunk 1 above)
Comment (polaroi8d) [reply web/crux/src/services/kratos.service.ts:153]: Talked in person.

**Review by polaroi8d — APPROVED** [review]
LGTM

**Review by m8vago — APPROVED** [review]

**Pull request merged**